integreat_cms/static/src/js/collapsible-boxes.ts

```diff
@@ -26,18 +29,35 @@ const toggleDashboardSection = async ({ target }: Event): Promise<void> => {
         setTimeout(() => {
             content.style.height = "auto";
         }, timeoutDuration);
+        collapsible.removeAttribute("data-collapsed");
     } else {
         // Set height to explicit numerical value instead of "auto" to enable continuous transition
         content.style.height = `${content.clientHeight}px`;
         // Use timeout to allow a smooth transition
         setTimeout(() => {
             content.style.height = "0px";
         });
+        collapsible.dataset.collapsed = "";
     }
+    // Store the currently collapsed boxes in a cookie
+    const collapsedBoxes = Array.from(document.querySelectorAll("[data-collapsed]:not([id=''])")).map((box) => box.id);
+    document.cookie = `collapsed-boxes=${JSON.stringify(collapsedBoxes)}`;
 };
 
 window.addEventListener("load", () => {
     document.querySelectorAll(".collapsible").forEach((node) => {
         node.addEventListener("click", toggleDashboardSection);
     });
+    // Restore all collapsed boxes
+    const collapsedBoxes = getCookie("collapsed-boxes");
+    if (collapsedBoxes) {
+        JSON.parse(collapsedBoxes).forEach((id: string) => {
+            const collapsible = document.querySelector(`.collapsible[id='${id}']`);
+            if (collapsible) {
+                collapsible.querySelector(".up-arrow").classList.add("hidden");
+                collapsible.querySelector(".down-arrow").classList.remove("hidden");
+                collapsible.parentNode.querySelector(".collapsible-content").classList.add("h-0");
```

Comment:
```suggestion
            const collapsible = document.querySelector(`.collapsible[id='${id}']`) as HTMLElement;
            if (collapsible) {
                collapsible.querySelector(".up-arrow").classList.add("hidden");
                collapsible.querySelector(".down-arrow").classList.remove("hidden");
                collapsible.parentNode.querySelector(".collapsible-content").classList.add("h-0");
                collapsible.dataset.collapsed = "";
```
And here we should probably also set the data attribute which is later used to store the cookie.